skt/runner.py

```diff
@@ -79,6 +92,17 @@ def getxml(self, replacements):
         return xml
 
     def getresultstree(self, jobid, logs=False):
+        """
+        Retrieve Beaker job results in Beaker's native XML format.
+
+        Args:
+            jobid: The beaker jobid
+            logs:  Set to 'True' to retrieve logs from beaker. The default is
+                   'False'.
```

Comment:
Could you please clarify a little what happens with `False`? I read this like the logs would then be retrieved from somewhere else, and was confused. I was probably thinking "logs" are the "results". Perhaps mentioning in the function description, that log inclusion into results is optional would help.

Reply:
Fixed.